extra_foam/algorithms/imageproc_py.py

```diff
@@ -7,13 +7,28 @@
 Copyright (C) European X-Ray Free-Electron Laser Facility GmbH.
 All rights reserved.
 """
+import numpy as np
+
 from .imageproc import (
     nanmeanImageArray, movingAvgImageData,
     imageDataNanMask, maskImageDataNan, maskImageDataZero,
-    correctGain, correctOffset, correctDsscOffset, correctGainOffset
+    correctGain, correctOffset, correctDsscOffset, correctGainOffset,
+    binPhotons
 )
 
 
+def bin_photons(data, adu_count, out=None):
+    rank = len(data.shape)
+    if rank != 2 and rank != 3:
```

Comment:
Nitpick: you could use `data.ndim` here, and `rank not in (2, 3)` prolly is more descriptive.

Reply:
Fixed in 34f83680.